### ISSUE TITLE
/usr/bin in /etc/paths obstructs path modification

### ISSUE THREAD

**Issue by ikonst** [timcharper/git_osx_installer#36]
The installer created /etc/paths.d/git, but then `/usr/libexec/path_helper` produced:

`PATH=" ... :/usr/bin: ... :/usr/local/git/bin"; export PATH;`

The `/usr/bin` above comes from /etc/paths. On my system, /etc/paths contained a bunch of standard system paths. Once /etc/paths is made empty (all lines removed), the /etc/paths.d paths can take precedence — `/usr/libexec/path_helper` now produces:

`PATH="/usr/local/git/bin:/usr/bin:/bin"; export PATH;`

(Yes, `/usr/bin` and `/bin` are still appended to the PATH, without appearing in /etc/paths nor /etc/paths.d, so having them in /etc/paths simply increased their precedence.)

**Comment by timcharper** [comment]
Yes, this is intentional. You need to move the system git out of the way, or symlink it directly in place.

It seems like there are a lot of issues caused around the system git and this installed; I've considered being more aggressive about making the installer the system git (maybe a hook to symlink itself into /usr/bin).
